src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  scroll-behavior: smooth;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  max-height: 60px;
+}
+
+header {
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+}
+
+nav {
+  padding-right: 50px;
+  align-items: center;
+  line-height: 60px;
+}
+
+.nav__links__list {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  max-height: 60px;
+}
+
+.logo {
+  padding-left: 40px;
+  object-fit: none;
```

Comment:
The `object-fit: none;` property in the `.logo` class might not be appropriate if the logo image needs to be resized or fit within a specific area. Consider using `object-fit: contain;` or `object-fit: cover;` depending on the design requirements.